contracts/polygon/connectors/gelato/aave-services/protection/events.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+
+contract Events {
+    event LogSubmitProtection(
+        address indexed dsa,
+        address indexed action,
+        uint256 wantedHealthFactor,
+        uint256 minimumHealthFactor,
+        bool isPermanent
+    );
+    event LogUpdateProtection(
+        address indexed dsa,
+        address indexed action,
+        uint256 wantedHealthFactor,
+        uint256 minimumHealthFactor,
+        bool isPermanent
+    );
+    event LogCancelProtection(address indexed dsa, address indexed action);
+}
```

Comment:
Can we also add an event like: `event LogCancelAndRevoke(address indexed dsa, address indexed action);`

Reply:
done